<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/DTU_red-blue.ico</normaloff>:/icons/DTU_red-blue.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1171</width>
      <height>661</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Debug_Connection_Main_vecrticalLayout" stretch="70,30">
     <item>
      <widget class="QTabWidget" name="Console_tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Console_tab">
        <attribute name="title">
         <string>Console</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>15</y>
           <width>785</width>
           <height>601</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="Console_Controls_verticalLayout" stretch="100,10">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTextEdit" name="Console_textEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>779</width>
              <height>523</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="midLineWidth">
             <number>0</number>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="ConsoleControls_horizontalLayout" stretch="3,1,20,1">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="Console_Export_exportConsole_pushButton">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
                <underline>false</underline>
                <strikeout>false</strikeout>
               </font>
              </property>
              <property name="text">
               <string>Export Console</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Console_Clear_pushButton">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>clear</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Console_lineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ConsoleSend_pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
                <underline>false</underline>
                <strikeout>false</strikeout>
               </font>
              </property>
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="TableView_tab">
        <attribute name="title">
         <string>Data Table View</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>11</y>
           <width>781</width>
           <height>601</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="TableView_Controls_verticalLayout" stretch="13,1">
          <item>
           <widget class="QTableView" name="Data_tableView">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>53</width>
              <height>116</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="TableViewControls_horizontalLayout">
            <property name="rightMargin">
             <number>550</number>
            </property>
            <item>
             <widget class="QPushButton" name="DataView_Export_exportConsole_pushButton">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
                <underline>false</underline>
                <strikeout>false</strikeout>
               </font>
              </property>
              <property name="text">
               <string>Export Console</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="DataView_Clear_pushButton">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="Debug_tabWidget">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Connections_tab">
        <attribute name="title">
         <string>Connections</string>
        </attribute>
        <widget class="QGroupBox" name="TCP_Connection_groupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>321</width>
           <height>280</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="title">
          <string>TCP Client</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QCheckBox" name="TCP_EnableManualInput_checkBox">
                <property name="text">
                 <string>Enter Manually</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="TCP_ManualInput_verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="TCP_ManualInput_IP_horizontalLayout">
                  <property name="spacing">
                   <number>22</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="TCP_ManualInput_IP_label">
                    <property name="text">
                     <string>IP</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="TCP_ManualInput_IP_lineEdit"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="TCP_ManualInput_Port_horizontalLayout">
                  <item>
                   <widget class="QLabel" name="TCP_ManualInput_Port_label">
                    <property name="font">
                     <font>
                      <pointsize>13</pointsize>
                      <weight>50</weight>
                      <bold>false</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>Port</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="TCP_ManualInput_Port_lineEdit"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="TCP_Select_verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="TCP_Select_IP_horizontalLayout" stretch="5,10">
                <property name="topMargin">
                 <number>15</number>
                </property>
                <item>
                 <widget class="QLabel" name="TCP_Select_IP_label">
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>IP</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="TCP_Select_IP_comboBox"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="TCP_Select_Port_horizontalLayout" stretch="5,10">
                <item>
                 <widget class="QLabel" name="TCP_Select_Port_label">
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Port</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="TCP_Select_Port_comboBox"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="TCP_Connect_pushButton">
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <weight>50</weight>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="Properties_tab">
        <attribute name="title">
         <string>Properties</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>11</y>
           <width>321</width>
           <height>601</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="Properties_tab_verticalLayout" stretch="6,4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableView" name="Properties_tableView"/>
          </item>
          <item>
           <widget class="QGroupBox" name="Properties_groupBox">
            <property name="title">
             <string notr="true"/>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="Commands_tab">
        <attribute name="title">
         <string>Commands</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>11</y>
           <width>321</width>
           <height>601</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QTreeView" name="Commands_treeView">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>5</number>
            </property>
            <property name="animated">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,5,2,2">
            <item>
             <widget class="QPushButton" name="Command_Save_pushButton">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Save </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Command_Import_pushButton">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Import</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Command_add_pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Command_delete_pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionSo">
   <property name="text">
    <string>connect</string>
   </property>
  </action>
  <action name="actionPorts">
   <property name="text">
    <string>Ports</string>
   </property>
  </action>
  <action name="actionBaud">
   <property name="text">
    <string>Baud</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
